cypress/e2e/give.cy.ts

```diff
@@ -0,0 +1,80 @@
+import { gqlQuery, injectWeb3 } from '../util';
+
+let circleId;
+
+context('Coordinape', () => {
+  before(() => {
+    Cypress.on('window:before:load', injectWeb3());
+    return gqlQuery({
+      circles: [
+        {
+          where: { organization: { name: { _eq: 'Fresh Open Epoch Admin' } } },
+        },
+        { id: true },
+      ],
+    }).then(q => {
+      circleId = q.circles[0].id;
+    });
+  });
+  it('can login and visit give', () => {
+    cy.visit(`/circles/${circleId}/give`);
+    cy.login();
+    cy.url({ timeout: 120000 }).should('include', '/give');
+    cy.contains('thank your teammates').click();
+  });
+  // it('can select teammates', () => {
```

Comment:
should we make this try the "+ Collaborator" thing when you hover on a row?

Reply:
did it